packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-option-multi/gux-option-multi.tsx

```diff
@@ -135,7 +146,7 @@ export class GuxOptionMulti {
         role="option"
         class={{
           'gux-active': this.active,
-          'gux-disabled': this.disabled,
+          'gux-disabled': this.disabled || this.hasDisabledParent(),
```

Comment:
This will probably also be needed in the `gux-option-icon` component.

Reply:
Added